src/history.h

```diff
@@ -33,47 +38,57 @@
 extern "C" {
 #endif
 
+/**
+ * A list of node addresses and sequence numbers
+ */
 typedef struct _DPS_NodeAddressList {
-    uint32_t sn;
-    DPS_NodeAddress addr;
-    struct _DPS_NodeAddressList* next;
+    uint32_t sn;                /**< A sequence number */
+    DPS_NodeAddress addr;       /**< A node address */
```

Comment:
in some locations you add extra white space to the separate the documentation from the variables being documented in other locations you don't. I am fine with both but I would try and be consistent.

for example compare this documentation to the documentation done for  _DPS_KeyStore struct in dps.h where no white space was added.

From this single commit. It looks like typically whitespace is added to line up the documentation making reading the code and documentation a little easier.

Reply:
Agreed, but I'll do this in another pull request